_includes/carbonStandard.html

```diff
@@ -42,10 +42,10 @@ <h3 class="pl-2 mb-0 text-xl font-medium text-white"><a class="text-inherit hove
       <h4 class="mt-4 mb-2 text-lg font-medium">Software</h4>
       {% include categoryItem.html bgColor="cat-u" icon="./assets/images/icons.svg#web" description="Off The Shelf including Open-Source Software" link="/categories/upstream#software" %}
 
-      <h4 class="mt-4 mb-2 text-lg font-medium">Hardware Manufacture, Transport and Installation</h4>
+      <h4 class="mt-4 mb-2 text-lg font-medium"><a class="text-inherit hover:text-inherit hover:opacity-80" href="/categories/upstream#hardware">Hardware Manufacture, Transport and Installations</a></h4>
```

Comment:
Should all of these headers be links to the categories?

Reply:
Yeah, I feel like this needs a discussion since the TCS diagram as it stands may need some tweaking for navigation to work consistently. For example, there's no 'On-premise' subheading for Category O, which is what I'd have liked to be able to link from for the new content i.e. usage carbon emissions (instead of linking generally to direct operational which includes generators).